ykrt/src/mt.rs

```diff
@@ -381,6 +285,8 @@ impl MT {
                     };
                     *hl = HotLocation::Compiled(tr);
                     loc.unlock();
+                    // LAURIE: Can we execute the JITted code now? Otherwise we have to interpret
```

Comment:
We *could* but that's a bit of extra complexity that I think we should avoid in this PR. Leaving a FIXME in to remind us to do it would be a good idea though.

Reply:
OK.

Reply:
Fixed added in 0df447df6.

You will also notice in 3bccbb593 that I had to make some of the tests run longer, as the JIT now takes one more iteration than is strictly necessary to enter JIT code.